package.js

```diff
@@ -1,12 +1,12 @@
 Package.describe({
     summary: "Login service for Stripe accounts using Stripe Connect",
     name: "chadkruser:meteor-accounts-stripe-connect",
-    version: "0.0.6",
+    version: "0.0.7",
     git: "https://github.com/chadokruse/meteor-accounts-stripe-connect.git",
 });
 
-Package.on_use(function(api) {
-    api.versionsFrom('METEOR@1.0');
+Package.onUse( function (api) {
+    api.versionsFrom('1.2');
```

Comment:
I've intended to make it work on 1.2

I'm not sure how to proceed about 1.0, what do you suggest?